src/bot.ts

```diff
@@ -392,23 +392,24 @@ a known bot info object.",
      */
     async start(options?: PollingOptions) {
         // Perform setup
+        const setup: Promise<void>[] = [];
```

Comment:
What about this?

```suggestion
        const setup = [];
```

Or this?

```suggestion
        const setup = new Array<Promise<void>>();
```

Reply:
The former does not type-check. The latter has a `new` too many. If anything, we can do
```suggestion
        const setup = Array<Promise<void>>();
```
but I prefer the current syntax because the code in the build output is more intuitive.